be/test/exprs/string_functions_test.cpp

```diff
@@ -252,6 +264,26 @@ TEST_F(StringFunctionsTest, null_or_empty) {
     delete context;
 }
 
+TEST_F(StringFunctionsTest, append_trailing_char_if_absent) {
+    ASSERT_EQ(StringVal("ac"), StringFunctions::append_trailing_char_if_absent(ctx,
+            StringVal("a"), StringVal("c")));
+
+    ASSERT_EQ(StringVal("c"), StringFunctions::append_trailing_char_if_absent(ctx,
+            StringVal("c"), StringVal("c")));
+
+    ASSERT_EQ(StringVal("123c"), StringFunctions::append_trailing_char_if_absent(ctx,
+            StringVal("123c"), StringVal("c")));
+
+    ASSERT_EQ(StringVal("c"), StringFunctions::append_trailing_char_if_absent(ctx,
+            StringVal(""), StringVal("c")));
+
+    ASSERT_EQ(StringVal::null(), StringFunctions::append_trailing_char_if_absent(ctx,
+            StringVal::null(), StringVal("c")));
+
+    ASSERT_EQ(StringVal::null(), StringFunctions::append_trailing_char_if_absent(ctx,
+            StringVal("a"), StringVal::null()));
```

Comment:
Should add test that second argument has multiple characters.

Reply:
ok，Done